docs/public/static/data/v48.0.0/expo-image-picker.json

```diff
@@ -1 +1 @@
e](#medialibrarypermissionresponse)."}]}]},"parameters":[{"name":"writeOnly","kind":32768,"comment":{"summary":[{"kind":"text","text":"Whether to request write or read and write permissions. Defaults to "},{"kind":"code","text":"`false`"}]},"type":{"type":"intrinsic","name":"boolean"},"defaultValue":"false"}],"type":{"type":"reference","typeArguments":[{"type":"reference","name":"MediaLibraryPermissionResponse"}],"name":"Promise","qualifiedName":"Promise","package":"typescript"}}]},{"name":"useCameraPermissions","kind":64,"signatures":[{"name":"useCameraPermissions","kind":4096,"comment":{"summary":[{"kind":"text","text":"Check or request permissions to access the camera.\nThis uses both "},{"kind":"code","text":"`requestCameraPermissionsAsync`"},{"kind":"text","text":" and "},{"kind":"code","text":"`getCameraPermissionsAsync`"},{"kind":"text","text":" to interact with the permissions."}],"blockTags":[{"tag":"@example","content":[{"kind":"code","text":"```ts\nconst [status, requestPermission] = ImagePicker.useCameraPermissions();\n```"}]}]},"parameters":[{"name":"options","kind":32768,"flags":{"isOptional":true},"type":{"type":"reference","typeArguments":[{"type":"intrinsic","name":"object"}],"name":"PermissionHookOptions"}}],"type":{"type":"tuple","elements":[{"type":"union","types":[{"type":"literal","value":null},{"type":"reference","name":"PermissionResponse"}]},{"type":"reference","typeArguments":[{"type":"reference","name":"PermissionResponse"}],"name":"RequestPermissionMethod"},{"type":"reference","typeArguments":[{"type":"reference","name":"PermissionResponse"}],"name":"GetPermissionMethod"}]}}]},{"name":"useMediaLibraryPermissions","kind":64,"signatures":[{"name":"useMediaLibraryPermissions","kind":4096,"comment":{"summary":[{"kind":"text","text":"Check or request permissions to access the media library.\nThis uses both "},{"kind":"code","text":"`requestMediaLibraryPermissionsAsync`"},{"kind":"text","text":" and "},{"kind":"code","text":"`getMediaLibraryPermissionsAsync`"},{"kind":"text","text":" to interact with the permissions."}],"blockTags":[{"tag":"@example","content":[{"kind":"code","text":"```ts\nconst [status, requestPermission] = ImagePicker.useMediaLibraryPermissions();\n```"}]}]},"parameters":[{"name":"options","kind":32768,"flags":{"isOptional":true},"type":{"type":"reference","typeArguments":[{"type":"reflection","declaration":{"name":"__type","kind":65536,"children":[{"name":"writeOnly","kind":1024,"flags":{"isOptional":true},"type":{"type":"intrinsic","name":"boolean"}}]}}],"name":"PermissionHookOptions"}}],"type":{"type":"tuple","elements":[{"type":"union","types":[{"type":"literal","value":null},{"type":"reference","name":"MediaLibraryPermissionResponse"}]},{"type":"reference","typeArguments":[{"type":"reference","name":"MediaLibraryPermissionResponse"}],"name":"RequestPermissionMethod"},{"type":"reference","typeArguments":[{"type":"reference","name":"MediaLibraryPermissionResponse"}],"name":"GetPermissionMethod"}]}}]}]}
```

Comment:
Can you revert docs data for SDK 48? We will not backport this change to SDK 48 😉